public/doc/openapi-3-v1.0.0.json

```diff
@@ -2635,7 +2635,7 @@
             "content": {
               "application/json": {
                 "schema": {
-                  "$ref": "#/components/schemas/DataDrivenFormSchema"
+                  "$ref": "#/components/schemas/ServicePlan"
```

Comment:
I'm thinking then that `DataDrivenForm` is no longer used in a path but we read it in for validation purposes?  I don't see an error in the SwaggerEditor along those lines so seems that is still valid.

Reply:
Yeah we use it when returning the result of PATCH modified, and as an example for the `create_json_schema` field for the ServicePlan schema. We could just inline it in both places or just say its an object with no example, which would you prefer?

Reply:
@syncrou updated to remove DDF definition.